test/e2e/common_suite_test.go

```diff
@@ -595,12 +595,13 @@ func doTestCreatePeerPodWithJob(t *testing.T, assert CloudAssert) {
 }
 
 func doTestCreatePeerPodAndCheckUserLogs(t *testing.T, assert CloudAssert) {
-	namespace := envconf.RandomName("default", 7)
-	podName := "user-pod"
-	imageName := "quay.io/confidential-containers/test-images:testuser"
-	pod := newPod(namespace, podName, podName, imageName, withRestartPolicy(v1.RestartPolicyOnFailure))
-	expectedPodLogString := "otheruser"
-	newTestCase(t, "UserPeerPod", assert, "Peer pod with user has been created").withPod(pod).withExpectedPodLogString(expectedPodLogString).withCustomPodState(v1.PodSucceeded).run()
+	// namespace := envconf.RandomName("default", 7)
+	// podName := "user-pod"
+	// imageName := "quay.io/confidential-containers/test-images:testuser"
+	// pod := newPod(namespace, podName, podName, imageName, withRestartPolicy(v1.RestartPolicyOnFailure))
+	// expectedPodLogString := "otheruser"
+	// newTestCase(t, "UserPeerPod", assert, "Peer pod with user has been created").withPod(pod).withExpectedPodLogString(expectedPodLogString).withCustomPodState(v1.PodSucceeded).run()
+	t.Skip("Skipping Test until Kata issue is Fixed")
```

Comment:
Can you add a reference to the kata issue here please

Reply:
Added the Issue link as comment